### DISCUSSION TITLE
Highlight current page in masthead (navigation bar)

### DISCUSSION THREAD

**Discussion by fortierq** [mmistakes/minimal-mistakes#2759] [Q&A] [answered]
Is it possible to highlight/underline the active page in the masthead (navigation bar)?

**Comment by jedick** [comment, marked as answer]
Yes, it's possible!

In `_includes/masthead.html` set the class attribute for the active `masthead__menu-item` link.

```html
{% if link.url == page.url %} class="active"{% endif %}
```

In `_layouts/default.html`, change an `include_cached` to `include`. The cached version of the masthead (as used in Minimal Mistakes) does not have the current `page.url`  - for me it was always the URL of the first blog post.

```html
{% include masthead.html %}
```

In `assets/css/main.scss` add CSS to style the active menu item.

```
.masthead__menu-item .active { font-weight: bold; }
```

Here is a commit to show these changes in context: https://github.com/jedick/jedick.github.io/commit/f4e2a78684459104997f32c722c56e7e35896b4f

I hope this helps anybody else coming across this question as I did. I was having no luck with AI answers about styling links in `nav__list`, which is for the *sidebar*, not the main menu or *masthead*.
